CPPLINT.cfg

```diff
@@ -49,8 +49,12 @@
 # 7. [-build/c++11], [+build/c++14]
 #
 #   This is to allow headers not approved for C++11 such as <chrono> etc. 
-# 
+#
+# 8. [-legal/copyright]
+#     Do not require a copyright header in each file
+#
+
 set noparent
-filter=-whitespace/braces,-runtime/references,-whitespace/indent,-whitespace/blank_line,-whitespace/newline,-build/include_subdir,-build/c++11,+build/c++14
-linelength=80
+filter=-whitespace/braces,-runtime/references,-whitespace/indent,-whitespace/blank_line,-whitespace/newline,-build/include_subdir,-build/c++11,+build/c++14,-legal/copyright
+linelength=127
```

Comment:
Gazebo uses line length 80. I've been adhering to this (and other Gazebo conventions which are different to both ROS and ArduPilot) as far as possible to be consistent with other plugins provided in the `gz-sim` repo.

Reply:
ok reverted then